cmd/fortifyExecuteScan.go

```diff
@@ -624,12 +629,11 @@ func populateMavenTranslate(config *fortifyExecuteScanOptions, classpath string)
 	translateList = append(translateList, make(map[string]interface{}))
 	translateList[0]["classpath"] = classpath
 
-	if len(config.Src) > 0 {
-		translateList[0]["src"] = config.Src
-	}
-	if len(config.Exclude) > 0 {
-		translateList[0]["exclude"] = config.Exclude
-	}
+	// "src" will be tokenized at " " in the end. This is probably fragile as it does not allow paths with spaces.
+	setTranslateEntryIfNotEmpty(translateList[0], "src", " ", config.Src,
+		[]string{"**/*.xml", "**/*.html", "**/*.jsp", "**/*.js", "**/src/main/resources/**/*", "**/src/main/java/**/*"})
```

Comment:
We now have the defaults in the code. Is there no way of keeping them in the metadata file?